cmd/ferretdb/ping.go

```diff
@@ -22,29 +22,35 @@ import (
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 	"go.uber.org/zap"
-
-	"github.com/FerretDB/FerretDB/internal/util/ctxutil"
 )
 
-// ping creates connection to FerretDB instance specified by the flags, and runs `ping` command against it.
-// The check is only executed if --setup-database flag is set.
-func ping() {
-	logger := setupLogger(cli.Log.Format, "")
-	checkFlags(logger)
+// Ping represents mongo driver client, which can be used to run `ping`
+// command against the FerretDB instance specified by cli flags.
+type Ping struct {
+	l *zap.Logger
+}
+
+// Probe executes ping queries to open listeners, and returns true if they succeed.
+// Any errors that occure are passed through Ping.l listener.
+//
+// It is only executed if --setup-database flag is set.
```

Comment:
Does the signature of this function `Probe(ctx) bool` have any significance related to kubernetes? Could you document that please? If it doesn't have any significance, I find it a bit strange to return boolean instead of an error indicating what failed.

Reply:
That was done in #4424. See how the signature is documented and how probes are used

Reply:
Oops thank you :see_no_evil: